jerry-core/parser/js/parser.h

```diff
@@ -29,8 +29,8 @@ typedef enum
 } jsp_status_t;
 
 void parser_set_show_instrs (bool);
-jsp_status_t parser_parse_script (const jerry_api_char_t *, size_t, const vm_instr_t **);
-jsp_status_t parser_parse_eval (const jerry_api_char_t *, size_t, bool, const vm_instr_t **, bool *);
-jsp_status_t parser_parse_new_function (const jerry_api_char_t **, const size_t *, size_t, const vm_instr_t **);
+jsp_status_t parser_parse_script (const jerry_api_char_t  *, size_t, const vm_instr_t * *);
+jsp_status_t parser_parse_eval (const jerry_api_char_t  *, size_t, bool, const vm_instr_t * *, bool  *);
+jsp_status_t parser_parse_new_function (const jerry_api_char_t * *, const size_t  *, size_t, const vm_instr_t * *);
```

Comment:
For the three functions above: Do we need the double spaces, and the space between `*` characters?